test-files/cmd/slow/ConflateCmdStatsAllDataTypesTest.sh.stdout

```diff
@@ -48,13 +48,13 @@ Polygons Conflatable by: hoot::PoiPolygonMatchCreator	10	8	0
 POIs Conflatable by: hoot::PoiPolygonMatchCreator	10	8	0	
 Features Conflatable by: hoot::PoiPolygonMatchCreator	20	16	0	
 Features Conflatable by: hoot::ScriptMatchCreator,PoiGeneric.js	11	10		
-Total Conflated Features	0	0	12	
-Percentage of Total Features Conflated	0	0	36.36363636363637	
+Total Conflated Features	0	0	13	
+Percentage of Total Features Conflated	0	0	38.23529411764706	
 Total Features Marked for Review	0	0	11	
-Percentage of Total Features Marked for Review	0	0	33.33333333333333	
+Percentage of Total Features Marked for Review	0	0	32.35294117647059	
 Total Reviews to be Made	0	0	8	
 Total Unmatched Features	25	22	21	
-Percentage of Total Features Unmatched	100	100	63.63636363636363	
+Percentage of Total Features Unmatched	100	100	61.76470588235294	
```

Comment:
Total conflated features went up, reviews down, and unmatched down.  That looks like success to me.